packages/core/src/defer/instructions.ts

```diff
@@ -328,7 +328,12 @@ export function ɵɵdefer(
     registry.add(ssrUniqueId, {lView, tNode, lContainer});
   }
 
-  const cleanupTriggersFn = () => invokeAllTriggerCleanupFns(lDetails, registry);
+  const cleanupTriggersFn = () => {
```

Comment:
nit: since the function is now responsible for cleaning up the registry, we can rename the function:
```suggestion
  const onLViewDestroy = () => {
```